docs/source/configuration/external_timelock_service_configs/timelock_server_config.rst

```diff
@@ -278,3 +278,29 @@ Note that because Timelock Server uses the OkHttp library, it is currently not c
    Thus, clients that are unable to use HTTP/2 may see a significant slowdown when the Timelock Server switches from an
    ``https`` connector to an ``h2`` connector. It may be possible to get around this by exposing multiple application
    connectors, though the AtlasDB team has not tested this approach.
+
+Non-Blocking Appender
```

Comment:
Ah, right. I'll add the label, but leave putting it in the release notes for #2215.